views/service.handlebars

```diff
@@ -0,0 +1,63 @@
+<section>
```

Comment:
Currently, this view looks to only be able to create a new service. We might want to use this view to show a service that isnt owned by the person who posted it as well.

Alternatively, we use a different view for doing that. No change required here but certainly something to think about and discuss.